.github/workflows/main.yml

```diff
@@ -97,7 +97,7 @@ jobs:
     - name: Install trgen
       shell: bash
       run: |
-        for i in trgen triconv trwdog trxml trxml2; do dotnet tool install -g $i --version 0.20.15; done
+        for i in trgen triconv trwdog trxml trxml2; do dotnet tool install -g $i --version 0.20.17; done
```

Comment:
Version update doesn't relate to PR title. Please extract these changes to another PR.